tools/scripts/clone.js

```diff
@@ -0,0 +1,45 @@
+const fs = require('fs')
+const path = require('path')
+
+const buildDir = path.resolve(__dirname, '../../build/')
+const builtFileNames = fs.readdirSync(buildDir)
+
+function prependSemicolon (fileName, text) {
+  const newText = ';' + text
```

Comment:
Clone works great, except this ends up prepending a double ;; to the output when running `npm run cdn:build:prod`.